src/corelibs/U2View/src/ov_msa/McaEditor.h

```diff
@@ -72,6 +72,10 @@ class U2VIEW_EXPORT McaEditor : public MaEditor {
 
     SequenceObjectContext *getReferenceContext() const;
 
+    QAction *getGotoSelectedReadAction() const {
```

Comment:
Call me picky, but I think, it's better either keep all functions in ".cpp" files, either have some kind of functions (for example, short ones, or "getters" and "setters"), which are all placed in ".h" files. In this case, all getters above are in ".cpp", but this one is here, so it's a bit confused.

Reply:
Trivial getters can be placed either to the header or source files: we have no strict code style about it.